cmd/argocd/commands/project.go

```diff
@@ -516,7 +536,12 @@ func printProjectLine(w io.Writer, p *v1alpha1.AppProject) {
 	default:
 		namespaceBlacklist = fmt.Sprintf("%d resources", len(p.Spec.NamespaceResourceBlacklist))
 	}
-	fmt.Fprintf(w, "%s\t%s\t%v\t%v\t%v\t%v\n", p.Name, p.Spec.Description, destinations, sourceRepos, clusterWhitelist, namespaceBlacklist)
+	if p.Spec.OrphanedResources == nil {
```

Comment:
what about `func (o *OrphanedResources) String()`?

Reply:
This struct has autogenernated String method, so I cannot override it. I agree it is bad to duplicate this logic, so I've created `formatOrphanedResources` function.